packages/ember-simple-auth/lib/simple-auth/mixins/application-route-mixin.js

```diff
@@ -37,13 +35,18 @@ var routeEntryComplete = false;
   @extends Ember.Mixin
   @static
 */
-export default Ember.Mixin.create({
+var ApplicationRouteMixin = Ember.Mixin.create({
```

Comment:
Can you change this back to `export default` so it's consistent with the other files again as the extra var isn't needed anymore?